Migrations/20191020221034_InitialCreate.cs

```diff
@@ -141,7 +141,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 });
 
             migrationBuilder.CreateTable(
-                name: "Ticket",
+                name: "Tickets",
```

Comment:
@mkorzecki To się samo tak zrobiło, po dodaniu migracji? Zaraz to będę sprawdzał, ale dziwnie to wygląda - to że są zmiany w pierwszej migracji, zamiast nowej migracji.